libraries/AP_Proximity/AP_Proximity_RPLidarA2.cpp

```diff
@@ -438,15 +434,15 @@ void AP_Proximity_RPLidarA2::parse_response_descriptor()
         }
         return;
     }
-    Debug(1, "Invalid response descriptor (%x %x)", _descriptor[0], _descriptor[1]);
+    Debug(2, "Invalid response descriptor (%x %x)", _descriptor[0], _descriptor[1]);
     _rp_state = rp_unknown;
 }
 
 void AP_Proximity_RPLidarA2::parse_response_data()
 {
     switch (_response_type){
         case ResponseType_SCAN:
-            Debug(3, "UART %02x %02x%02x %02x%02x", payload[0], payload[2], payload[1], payload[4], payload[3]); //show HEX values
+            Debug(0, "uart: %02x %02x %02x %02x %02x", payload[0], payload[1], payload[2], payload[3], payload[4]); //show HEX values
```

Comment:
Why did you change the order and the grouping? It was in a human friendly form.

Reply:
I was looking at the raw data to track the errors, thus the "human friendly" form wasn't helpful for me.